lib/models/requests.js

```diff
@@ -60,7 +60,7 @@ module.exports.newRequest = function(isFromIrc) {
 
     // expose an error handler to prevent defer boilerplate leaking everywhere
     request.errFn = function(err) {
-        if (err.stack) {
+        if (err.stack && err.message !== module.exports.ERR_VIRTUAL_USER) {
```

Comment:
I'm starting to reject with actual errors, and `ERR_VIRTUAL_USER` is one of them.